src/haggadah/routes.cljs

```diff
@@ -10,7 +10,12 @@
    [haggadah.events :as events]))
 
 
-
+(defn run-events
+  "Pre: takes a collection of events
+  Post: dispatches every event in the collection"
+  [events]
+  (doseq [event events]
+    (re-frame/dispatch event)))
```

Comment:
I thought there's a `dispatch-n` or something like that...

Reply:
There is a `dispatch-n`, but it was deprecated for the `:fx` key in a `reg-event-fx` handler. I'm using this to dispatch all the events in collection without using a handler. Would it be better to make a handler which solely runs all the events passed to it?

Reply:
But the doc says
> this effect is deprecated in favour of using :fx with multiple :dispatch tuples.